Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_8.yml

```diff
@@ -1274,6 +1283,66 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    scriptarguments:
+      Account:
+        complex:
+          root: Account
+          transformers:
+          - operator: uniq
+      CriticalEndpoints:
+        simple: admin
+      CriticalGroups:
+        simple: admins,administrators
```

Comment:
why this is being passed manually ? and not as an input or thorough the parent PB inputs?

Reply:
These are the same values as before. They're configured in the Calculate Severity playbook since we still don't have input groupings. The reason we see this in the PR is because we've changed the DBotScore inputs of Calculate Severity - Generic v2, so it's just adding all the inputs again.

Reply:
The reason why I mentioned that is because it can change PB logic for those who configured this input through the parent PB `Calculate Severity - Generic v2`. By mentioning it here, it will overwrite what has been mentioned there. If you are comparable with this change - ignore this comment, otherwise, just remove these 2 inputs from here.

Reply:
You're correct, I think it will overwrite. I'll fix it obviously :)